newIDE/app/src/stories/componentStories/AnnouncementsFeed/AnnouncementsFeed.stories.js

```diff
@@ -48,8 +53,9 @@ export const ErrorLoadingAnnouncements = () => {
     <AnnouncementsFeedContext.Provider
       value={{
         announcements: null,
+        promotions: null,
         error: new Error('Fake error'),
-        fetchAnnouncements: action('fetchAnnouncements'),
+        fetchAnnouncementsAndPromotions: action('fetchAnnouncements'),
```

Comment:
```suggestion
        fetchAnnouncementsAndPromotions: action('fetchAnnouncementsAndPromotions'),
```
And in the other places also